local_costmap:
    update_frequency: 1
    publish_frequency: 0
    static_map: false
    rolling_window: true
    width: 8.0
    height: 8.0
    track_unknown_space: false
